25_exceptions_lab/01.py

```diff
@@ -1,7 +1,22 @@
-"""
-Write a python program that takes numbers in a loop
-and for each number prints its square root
-If value is negative or not a number show 
-a warning and keep reading values
-"""
-
+#commited by Shlomit
+
+import numbers
+import sys
+import math
+def sqrt(n):

```

Comment:
Don't do that- Python already has a sqrt function that already raises exceptions. Just use existing and handle the exception in the bottom loop